samples/sensor/hts221/prj.conf

```diff
@@ -3,3 +3,5 @@ CONFIG_I2C=y
 CONFIG_SENSOR=y
 CONFIG_HTS221=y
 CONFIG_HTS221_TRIGGER_NONE=y
+#CONFIG_HTS221_TRIGGER_GLOBAL_THREAD=y
+#CONFIG_HTS221_TRIGGER_OWN_THREAD=y
```

Comment:
Leftover?

Could add a comment explaining why these commented-out assignments are here otherwise.

Reply:
I suppose, though it doesn't really seem worth it.  The reason is that anybody using these to verify the sensor or learn how it functions could want to experiment with the three ways to get observations out of it.

Reply:
I don't understand why the commented-out assignments are there, so a comment is worth it.

Is it just a "you might want to try out these options too"? Ought to explain it in that case.

Reply:
Ought to mention that they're mutually exclusive too. Something like

    You might want to try these ways of yada yada, because yada yada. Note that
    these options are mutually exclusive.

Reply:
Forget it, I'll just add them in locally when I need them.